feat(serverless): containers & functions improve quickstart (#1562)
scaleway/docs-content: serverless/containers/quickstart.mdx

A container is a package of software that includes all dependencies: code, runtime, configuration, and system libraries so that the software can run on any host system. The Scaleway Containers platform provides you with containerized stateless applications deployed in the cloud. You can use your favorite technologies such as Django or Ruby On Rails for your containers. We show you how to create a containers namespace, deploy a container, and delete it when you are done.

<Message type="tip">

>   This page explains how to deploy a containers using Scaleway Console but for advanced usage (ex: larger project with dependencies, automation, multiple containers...) there are a [lot of ways to deploy containers](/serverless/containers/reference-content/deploy-container/).

Reviewer: ```suggestion
  This page explains how to deploy a containers using Scaleway Console but for advanced usage (ex: larger project with dependencies, automation, multiple containers...) [several ways to deploy containers](/serverless/containers/reference-content/deploy-container/) are available.
```

>   This page explains how to deploy a containers using Scaleway Console but for advanced usage (ex: larger project with dependencies, automation, multiple containers...) [several ways to deploy containers](/serverless/containers/reference-content/deploy-container/) are available.

Reviewer 2: ```suggestion
  This page explains how to deploy containers using Scaleway Console but for advanced usage (ex: larger projects with dependencies, automation, multiple containers...) [several ways to deploy containers](/serverless/containers/reference-content/deploy-container/) are available.
```

>   For real world examples checkout our [scaleway/serverless-examples repository](https://github.com/scaleway/serverless-examples).

Reviewer 2: ```suggestion
  For real-world examples check out our [scaleway/serverless-examples repository](https://github.com/scaleway/serverless-examples).
```
